cs/cli/Program.cs

```diff
@@ -36,6 +37,17 @@ public static void InitOptions(Exchange instance, IEnumerable<string> args)
 
     public static void SetCredentials(Exchange instance)
     {
+         JObject localKeys = null;
+        try
+        {
+            var jsonText = File.ReadAllText("keys.local.json");
+            localKeys = JsonConvert.DeserializeObject<JObject>(jsonText);
+        }
+        catch (JsonException je)
+        {
```

Comment:
![image](https://github.com/ccxt/ccxt/assets/43336371/4cf8d343-2d97-4959-905c-e9a3593090a6)
 we shouldn't throw if the file does not exist because we might be reading from the environment